octobot_trading/personal_data/orders/decimal_order_adapter.py

```diff
@@ -104,6 +107,29 @@ def decimal_adapt_order_quantity_because_price(limiting_value, max_value, price,
     return orders
 
 
+def decimal_adapt_order_quantity_because_fees(
+    exchange_manager, symbol: str, order_type: enums.TraderOrderType, quantity: decimal.Decimal, price: decimal.Decimal,
+    taker_or_maker: enums.ExchangeConstantsMarketPropertyColumns, side: enums.TradeOrderSide,
+    quote_available_funds: decimal.Decimal
+) -> decimal.Decimal:
+    if side == enums.TradeOrderSide.BUY:
+        # only buy orders are affected
+        computed_fee = exchange_manager.exchange.get_trade_fee(
+            symbol, order_type, quantity, price, taker_or_maker.value
+        )
+        quote = commons_symbols.parse_symbol(symbol).quote
+        # if fee paid in quote, ensure enough remaining quote asset in available portfolio
+        if currency_fee := personal_data.get_fees_for_currency(computed_fee, quote):
+            fees_in_base = currency_fee / price if price else constants.ZERO
```

Comment:
I don't think we should devide by zero when default value

Reply:
indeed !